client/components/team/imports/team-helpers.js

```diff
@@ -3,28 +3,34 @@ import { withTracker } from 'meteor/react-meteor-data';
 
 export const DIVISION_TYPES = [
   {
-    text: 'WWU Student',
-    value: 'wwu-student',
+    text: 'WWU Students',
+    value: 'wwu-students',
     wristBandColor: "Green",
     description: "All team members must be currently enrolled at WWU (undergrad or grad)."
   },
   {
     text: 'WWU Alumni',
     value: 'wwu-alumni',
     wristBandColor: "Blue",
-    description: "At least half of team members must be WWU Alumni."
+    description: "At least half of team members must be WWU Alumni"
   },
   {
-    text: 'High School',
-    value: 'highschool',
+    text: 'Postsecondary Students',
+    value: 'postsecondary',
     wristBandColor: "Purple",
-    description: "All team members must be currently enrolled in high school. Exception: One adult chaperone per team may register as a team member."
+    description: "All team members must be currently enrolled in college (undergrad or grad), technical school, running start. Mix and match-team members from same or different schools."
```

Comment:
Should it say "technical school, **or** running start"?

Reply:
Good catch!